app/experimenter/experiments/tests/serializers/test_timeline_population.py

```diff
@@ -0,0 +1,122 @@
+from django.test import TestCase
+from rest_framework import serializers
+
+from experimenter.experiments.tests.factories import (
+    ExperimentFactory,
+    LocaleFactory,
+    CountryFactory,
+)
+
+from experimenter.experiments.serializers.timeline_population import (
+    ExperimentTimelinePopSerializer,
+    CountrySerializerMultiSelect,
+    LocaleSerializerMultiSelect,
+)
+from experimenter.experiments.constants import ExperimentConstants
+from experimenter.experiments.tests.mixins import MockRequestMixin
+
+
+class TestLocaleSerializerMultiSelect(serializers.ModelSerializer):
+
+    def test_serializer_outputs_expected_schema(self):
+        locale = LocaleFactory.create()
+        serializer = LocaleSerializerMultiSelect(locale)
+
+        self.assertEqual(serializer.data, {"value": locale.code, "label": locale.name})
+
+
+class TestCountrySerializerMultiSelect(serializers.ModelSerializer):
+
+    def test_serializer_outputs_expected_schema(self):
+        country = CountryFactory.create()
+        serializer = CountrySerializerMultiSelect(country)
+
+        self.assertEqual(serializer.data, {"value": country.code, "label": country.name})
+
+
+class TestExperimentTimelinePopSerializer(MockRequestMixin, TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        self.experiment = ExperimentFactory.create(
+            type=ExperimentConstants.TYPE_PREF,
+            locales=[LocaleFactory.create()],
+            countries=[CountryFactory.create()],
+        )
+
+    def test_serializer_outputs_expected_schema(self):
+
+        serializer = ExperimentTimelinePopSerializer(self.experiment)
+
+        self.assertEqual(
+            serializer.data["proposed_start_date"],
+            self.experiment.proposed_start_date.strftime("%Y-%m-%d"),
+        )
+        self.assertEqual(
+            serializer.data["firefox_channel"], self.experiment.firefox_channel
+        )
+        self.assertEqual(serializer.data["platform"], self.experiment.platform)
```

Comment:
Why does this test only check these three fields?

Reply:
I'll make it check all of them.